deploy/helm/sumologic/README.md

```diff
@@ -55,6 +55,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `sumologic.logs.multiline.additional`                                                 | List of additional conditions and expressions to match first line of multiline logs. See [Multiline](/docs/collecting-container-logs.md#conditional-multiline-log-parsing) for more information.                                                                                                                                                                     | `[]`                                                                                                                                                                                 |
 | `sumologic.logs.systemd.enabled`                                                      | Enable collecting systemd logs from Kubernets nodes.                                                                                                                                                                                                                                                                                                                 | `true`                                                                                                                                                                               |
 | `sumologic.logs.systemd.units`                                                        | List of systemd units to collect logs from.                                                                                                                                                                                                                                                                                                                          | See [values.yaml]                                                                                                                                                                    |
+| `sumologic.logs.container.keep_time_attribute`                                        | When set to `true`, preserves the `time` attribute, which is a string representation of the `timestamp` attribute.                                                                                                                                                                                                                                                   | `false`                                                                                                                                                                              |
```

Comment:
I know this has been already messed up, but wouldn't we like to have this list sorted alphabetically?

Reply:
Yes, ideally I'd expect to have this list sorted.